Sources/Core/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
   }
 #endif
 
+  opcode.clear();
```

Comment:
Sure why not. I mostly wanted to check if there was a bug you were fixing
or something :)

On Friday, August 12, 2016, Francis Ricci <notifications@github.com> wrote:

> In Sources/Core/ARM/SoftwareBreakpointManager.cpp
> <https://github.com/facebook/ds2/pull/422#discussion_r74682915>:
>
> > @@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
> >    }
> >  #endif
> >
> > +  opcode.clear();
>
> Yeah, not using it just assumes the caller passes a clean argument. Which
> is fine, but I also don't think you incur any cost from clearing an empty
> vector. So better to prevent future bugs?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/422/files/50a5cbdc6a795f7383c093429619b7c05ce45b12..7cdfc54368119ccdfb434303997ba7f05611c54e#r74682915>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbOi_owKnmSSFs4dKmh56LEyGuQcxks5qfWZJgaJpZM4JjTCI>
> .
>


-- 
Stephane Sezer